tests/platform_tests/test_platform_info.py

```diff
@@ -439,7 +439,7 @@ def test_thermal_control_fan_status(duthosts, rand_one_dut_hostname, mocker_fact
         single_fan_mocker = mocker_factory(duthost, 'SingleFanMocker')
         time.sleep(THERMAL_CONTROL_TEST_WAIT_TIME)
 
-        _fan_log_supported = duthost.command('docker exec pmon grep -E "{}" /usr/bin/thermalctld'\
+        _fan_log_supported = duthost.command('docker exec pmon grep -E "{}" /usr/local/bin/thermalctld'\
```

Comment:
Thanks for catching and fixing! However, these tests are expected to run on older versions of SONiC, before the path was changed. Thus I suggest something like the following:

```python
        if "201811" in dut.os_version or "201911" in dut.os_version:
            THERMALCTLD_PATH = '/usr/bin/thermalctld'
        else:
            THERMALCTLD_PATH = '/usr/local/bin/thermalctld'

        _fan_log_supported = duthost.command('docker exec pmon grep -E "{}" {}'
                .format(LOG_EXPECT_INSUFFICIENT_FAN_NUM_RE, THERMALCTLD_PATH), module_ignore_errors=True)
```

Reply:
Hi @jleveque 
Can you please review new